### PR TITLE
Feature/redux setup

### PR DESCRIPTION
**Pull request by Menyten**
React, redux, saga setup etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# Lan-Time
```
Comment (BrandtNick) [inline README.md]: Why remove this file?

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,26 +1,6 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```
Comment (BrandtNick) [inline frontend/src/App.js]: Single quotes on all imports.

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,26 +1,6 @@
...
+import Views from "./views/index.js";
```
Comment (BrandtNick) [inline frontend/src/App.js]: Single quotes on all imports.

File: frontend/src/components/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (BrandtNick) [inline frontend/src/components/index.js]: Single quotes on all imports.

File: frontend/src/components/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+import Start from "../views/Start";
```
Comment (BrandtNick) [inline frontend/src/components/index.js]: Single quotes on all imports.

File: frontend/src/state/reducers/rootReducer.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import { page } from "./pageReducer";
```
Comment (BrandtNick) [inline frontend/src/state/reducers/rootReducer.js]: Single quotes on all imports.

File: frontend/src/state/store/configureStore.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
+import { connectRoutes } from 'redux-first-router'
+import { 
+  createStore, 
+  applyMiddleware, 
+  compose
+ } from 'redux';
+import createSagaMiddleware from "redux-saga"
```
Comment (BrandtNick) [inline frontend/src/state/store/configureStore.js]: Single quotes on all imports.

File: frontend/src/state/store/configureStore.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+import { composeWithDevTools } from 'redux-devtools-extension'
+
+import rootReducer from '../reducers/rootReducer'
+import { routePaths } from '../../views'
+import { rootSaga } from "./rootSaga"
```
Comment (BrandtNick) [inline frontend/src/state/store/configureStore.js]: Single quotes on all imports.

File: frontend/src/state/store/configureStore.js [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+const {
+  middleware: routerMiddleware,
+  enhancer: routerEnhancer,
+  initialDispatch
+} = connectRoutes(routePaths, { initialDispatch: false });
+
+/**
+ * Root Store
+ */
+export default function configureStore() {
```
Comment (BrandtNick) [inline frontend/src/state/store/configureStore.js]: arrow function

File: frontend/src/views/Start/Startpage.js [hunk 10]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (BrandtNick) [inline frontend/src/views/Start/Startpage.js]: Single quotes on all imports.

File: frontend/src/views/index.js [hunk 11]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
```
Comment (BrandtNick) [inline frontend/src/views/index.js]: Single quotes on all imports.

File: frontend/src/views/index.js [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+import * as components from "../components/index";
```
Comment (BrandtNick) [inline frontend/src/views/index.js]: Single quotes on all imports.

File: frontend/src/views/index.js [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+import { connect } from "react-redux";
```
Comment (BrandtNick) [inline frontend/src/views/index.js]: Single quotes on all imports.

File: frontend/src/views/index.js [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+
+export const routePaths = {
+  "HOME": "/"
```
Comment (BrandtNick) [inline frontend/src/views/index.js]: Single quotes

File: frontend/src/constants.js [hunk 15]
```diff
@@ -0,0 +1 @@
+export const HOME = "HOME";
```
Comment (BrandtNick) [inline frontend/src/constants.js]: Single quotes.

File: frontend/src/index.js [hunk 16]
```diff
@@ -3,8 +3,16 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import configureStore from '../src/state/store/configureStore';
+import { Provider } from "react-redux";
```
Comment (BrandtNick) [inline frontend/src/index.js]: Single quotes

File: frontend/src/state/store/configureStore.js [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+  createStore, 
+  applyMiddleware, 
+  compose
+ } from 'redux';
+import createSagaMiddleware from 'redux-saga'
+import { composeWithDevTools } from 'redux-devtools-extension'
+
+import rootReducer from '../reducers/rootReducer'
+import { routePaths } from '../../views'
+import { rootSaga } from "./rootSaga"
```
Comment (BrandtNick) [inline frontend/src/state/store/configureStore.js]: single quotes

**Comment by Punaaa** [comment]
Alla dina reviews är lösta nu nick.

**Pull request merged**